src/components/Details/Details.js

```diff
@@ -29,40 +29,39 @@ const ALBUMS = gql`
   // }`
 
 const DetailsModal = () => {
+
   const discogsLink = 'https://www.discogs.com/James-Brown-The-Payback/master/33990';
   const { loading, error, data } = useQuery(ALBUMS);
 
   if (loading) return <p>Loading...</p>;
   if (error) return <p>Error :(</p>;
-
-  console.log(data)
   
   return (
     <div className="modal-container">
-      <section className="modal">
+      <section className="modal" data-cy="modal">
         <article className="box left">
-          <img className="album-cover shadow" src={data.album.coverImage} alt={`${data.album.title} album cover`}/>
+          <img data-cy="album-cover" className="album-cover shadow" src={data.album.coverImage} alt={`${data.album.title} album cover`}/>
```

Comment:
lets make sure the cars and modal classes don't interfere with each other once all merged